styles/myStory.css

```diff
@@ -0,0 +1,59 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  background-image: url("/images/image8.jpg");
+  background-size: cover;
+  background-color: #f0f0f0;
+}
+
+.overlay-text {
+  position: absolute;
+  color: white;
+  font-family: var(--paragraph-sans);
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.overlay-text p {
+  margin-bottom: 50px;
+  margin: 0px;
+  text-transform: uppercase;
+  text-align: center;
+}
+.text-header {
+  font-size: 68px;
+}
+.text-description {
+  padding-top: 20px;
+  font-size: 16px;
+}
+
+.myStory-content-wrapper {
+  background-color: white;
+  max-width: 1500px;
+}
+.myStory-inner-content {
+  padding: 40px 20vw;
```

Comment:
![CleanShot 2024-11-06 at 15 46 21](https://github.com/user-attachments/assets/3b605799-b395-4b95-88ea-68fe6f3ae9b9)
This is not centered 😆

Reply:
I moved the max-width:1500px and margin: 0 auto to inner-content to fix this